config/locales/es.yml

```
@@ -0,0 +1,81 @@
+# Files in the config/locales directory are used for internationalization
+# and are automatically loaded up by Rails. If you want to use locales other
+# than English, add the necessary files in this directory.
+#
+# To use the locales, use `I18n.t`:
+#
+#     I18n.t 'hello'
+#
+# In views, this is aliased to just `t`:
+#
+#     <%= t('hello') %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information from config/locales/es.yml.
+#
+# The following keys must be escaped, otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   'true': 'foo'
+#
+# To learn more, please read the Rails Internationalization guide
+# available at https://guides.rubyonrails.org/i18n.html.
+
+es:
+  language_name: Spanish
```

@taicoding would be good to provide Spanish language name in Spanish
```suggestion
  language_name: Español
```

---

You are right. I make the update